### ISSUE TITLE
fatal error: 'Flutter\Flutter.h' file not found #import <Flutter\Flutter.h> - IOS

### ISSUE THREAD

**Issue by jbryanh** [flutter/flutter#76520]
## Steps to Reproduce

I am submitting this as requested by @jmagman in [#70895](https://github.com/flutter/flutter/issues/70895#issuecomment-780085361) 

1. Run `flutter create bug`.
2. Update the files as follows: ... 

3. ... 
Working on one application on Sunday, actually compiled and uploaded to ios store.  closed Android Studio.
Monday Morning, opened Android Studio and opened a project for a class...very simple.  (I do not autoload last project, but choose from a list of recent projects).

Got the following fatal error:
 In file included from /Users/bryanhunsberger/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/path_provider-1.6.27/ios/Classes/FLTPathProviderPlugin.m:5:
               /Users/bryanhunsberger/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/path_provider-1.6.27/ios/Classes/FLTPathProviderPlugin.h:5:9: fatal error:
               'Flutter/Flutter.h' file not found
               #import <Flutter/Flutter.h>

I do not have this plugin in this application...it's basically a starter app.  However, my application from Sunday does.

**Expected results:** 
Expect a simple program to compile to emulator.

**Actual results:** 
Note that my logs are too long to include in entirety, so I just included with reference to path_provider and below.  If you want to top end, let me know and I can email or whatever.
<details>
  <summary>Logs</summary>

```
/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/path_pro
                   vider.build/path_provider-generated-files.hmap

-I/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/path_p
                   rovider.build/path_provider-own-target-headers.hmap

-I/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/path_p
                   rovider.build/path_provider-all-non-framework-target-headers.hmap -ivfsoverlay

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/path_pro
                   vider.build/all-product-headers.yaml -iquote

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/path_pro
                   vider.build/path_provider-project-headers.hmap
                   -I/Users/bryanhunsberger/AppBreweryProjectsAll/app_brewery_projects_all/build/ios/Debug-iphonesimulator/path_provider/include

-I/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/path_p
                   rovider.build/DerivedSources-normal/x86_64

-I/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/path_p
                   rovider.build/DerivedSources/x86_64

-I/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/path_p
                   rovider.build/DerivedSources -F/Users/bryanhunsberger/AppBreweryProjectsAll/app_brewery_projects_all/build/ios/Debug-iphonesimulator/path_provider
-include
                   /Users/bryanhunsberger/AppBreweryProjectsAll/app_brewery_projects_all/ios/Pods/Target\ Support\ Files/path_provider/path_provider-prefix.pch -MMD -MT
dependencies
                   -MF

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/path_pro
                   vider.build/Objects-normal/x86_64/FLTPathProviderPlugin.d --serialize-diagnostics

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/path_pro
                   vider.build/Objects-normal/x86_64/FLTPathProviderPlugin.dia -c
                   /Users/bryanhunsberger/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/path_provider-1.6.27/ios/Classes/FLTPathProviderPlugin.m -o

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/path_pro
                   vider.build/Objects-normal/x86_64/FLTPathProviderPlugin.o
               In file included from /Users/bryanhunsberger/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/path_provider-1.6.27/ios/Classes/FLTPathProviderPlugin.m:5:
               /Users/bryanhunsberger/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/path_provider-1.6.27/ios/Classes/FLTPathProviderPlugin.h:5:9: fatal error:
               'Flutter/Flutter.h' file not found
               #import <Flutter/Flutter.h>
                       ^~~~~~~~~~~~~~~~~~~
               1 error generated.
           
               ProcessInfoPlistFile
               /Users/bryanhunsberger/AppBreweryProjectsAll/app_brewery_projects_all/build/ios/Debug-iphonesimulator/audioplayers/audioplayers.framework/Info.plist
               /Users/bryanhunsberger/AppBreweryProjectsAll/app_brewery_projects_all/ios/Pods/Target\ Support\ Files/audioplayers/audioplayers-Info.plist (in target
'audioplayers'
               from project 'Pods')
                   cd /Users/bryanhunsberger/AppBreweryProjectsAll/app_brewery_projects_all/ios/Pods
                   builtin-infoPlistUtility /Users/bryanhunsberger/AppBreweryProjectsAll/app_brewery_projects_all/ios/Pods/Target\ Support\
Files/audioplayers/audioplayers-Info.plist
                   -producttype com.apple.product-type.framework -expandbuildsettings -format binary -platform iphonesimulator -o
                   /Users/bryanhunsberger/AppBreweryProjectsAll/app_brewery_projects_all/build/ios/Debug-iphonesimulator/audioplayers/audioplayers.framework/Info.plist
           
               CompileSwiftSources normal x86_64 com.apple.xcode.tools.swift.compiler (in target 'audioplayers' from project 'Pods')
                   cd /Users/bryanhunsberger/AppBreweryProjectsAll/app_brewery_projects_all/ios/Pods
                   export DEVELOPER_DIR\=/Applications/Xcode.app/Contents/Developer
                   export SDKROOT\=/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.2.sdk
                   /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swiftc -incremental -module-name audioplayers -Onone
-enable-batch-mode
                   -enforce-exclusivity\=checked

@/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopl
                   ayers.build/Objects-normal/x86_64/audioplayers.SwiftFileList -DDEBUG -D COCOAPODS -sdk
                   /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.2.sdk -target
x86_64-apple-ios8.0-simulator -g
                   -module-cache-path /Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/ModuleCache.noindex -Xfrontend -serialize-debugging-options -enable-testing
                   -swift-version 5 -I /Users/bryanhunsberger/AppBreweryProjectsAll/app_brewery_projects_all/build/ios/Debug-iphonesimulator/audioplayers -F
                   /Users/bryanhunsberger/AppBreweryProjectsAll/app_brewery_projects_all/build/ios/Debug-iphonesimulator/audioplayers -c -j4 -output-file-map

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/Objects-normal/x86_64/audioplayers-OutputFileMap.json -parseable-output -serialize-diagnostics -emit-dependencies -emit-module
-emit-module-path

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/Objects-normal/x86_64/audioplayers.swiftmodule -Xcc

-I/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiop
                   layers.build/swift-overrides.hmap -Xcc -iquote -Xcc

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/audioplayers-generated-files.hmap -Xcc

-I/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiop
                   layers.build/audioplayers-own-target-headers.hmap -Xcc

-I/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiop
                   layers.build/audioplayers-all-non-framework-target-headers.hmap -Xcc -ivfsoverlay -Xcc

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/all-product-headers.yaml -Xcc -iquote -Xcc

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/audioplayers-project-headers.hmap -Xcc
                   -I/Users/bryanhunsberger/AppBreweryProjectsAll/app_brewery_projects_all/build/ios/Debug-iphonesimulator/audioplayers/include -Xcc

-I/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiop
                   layers.build/DerivedSources-normal/x86_64 -Xcc

-I/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiop
                   layers.build/DerivedSources/x86_64 -Xcc

-I/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiop
                   layers.build/DerivedSources -Xcc -DPOD_CONFIGURATION_DEBUG\=1 -Xcc -DDEBUG\=1 -Xcc -DCOCOAPODS\=1 -emit-objc-header -emit-objc-header-path

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/Objects-normal/x86_64/audioplayers-Swift.h -import-underlying-module -Xcc -ivfsoverlay -Xcc

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/unextended-module-overlay.yaml -working-directory /Users/bryanhunsberger/AppBreweryProjectsAll/app_brewery_projects_all/ios/Pods
               CompileSwift normal x86_64
/Users/bryanhunsberger/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/audioplayers-0.17.3/ios/Classes/NotificationsHandler.swift (in
               target 'audioplayers' from project 'Pods')
                   cd /Users/bryanhunsberger/AppBreweryProjectsAll/app_brewery_projects_all/ios/Pods
                   /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swift -frontend -c -primary-file
                   /Users/bryanhunsberger/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/audioplayers-0.17.3/ios/Classes/NotificationsHandler.swift
                   /Users/bryanhunsberger/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/audioplayers-0.17.3/ios/Classes/SwiftAudioplayersPlugin.swift
                   /Users/bryanhunsberger/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/audioplayers-0.17.3/ios/Classes/Utils.swift
                   /Users/bryanhunsberger/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/audioplayers-0.17.3/ios/Classes/WrappedMediaPlayer.swift -emit-module-path

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/Objects-normal/x86_64/NotificationsHandler\~partial.swiftmodule -emit-module-doc-path

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/Objects-normal/x86_64/NotificationsHandler\~partial.swiftdoc -emit-module-source-info-path

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/Objects-normal/x86_64/NotificationsHandler\~partial.swiftsourceinfo -serialize-diagnostics-path

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/Objects-normal/x86_64/NotificationsHandler.dia -emit-dependencies-path

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/Objects-normal/x86_64/NotificationsHandler.d -emit-reference-dependencies-path

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/Objects-normal/x86_64/NotificationsHandler.swiftdeps -target x86_64-apple-ios8.0-simulator -enable-objc-interop -sdk
                   /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.2.sdk -I
                   /Users/bryanhunsberger/AppBreweryProjectsAll/app_brewery_projects_all/build/ios/Debug-iphonesimulator/audioplayers -F
                   /Users/bryanhunsberger/AppBreweryProjectsAll/app_brewery_projects_all/build/ios/Debug-iphonesimulator/audioplayers -enable-testing -g
-import-underlying-module
                   -module-cache-path /Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/ModuleCache.noindex -swift-version 5 -enforce-exclusivity\=checked -Onone
-D DEBUG -D
                   COCOAPODS -serialize-debugging-options -Xcc -working-directory -Xcc /Users/bryanhunsberger/AppBreweryProjectsAll/app_brewery_projects_all/ios/Pods
                   -enable-anonymous-context-mangled-names -Xcc

-I/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiop
                   layers.build/swift-overrides.hmap -Xcc -iquote -Xcc

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/audioplayers-generated-files.hmap -Xcc

-I/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiop
                   layers.build/audioplayers-own-target-headers.hmap -Xcc

-I/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiop
                   layers.build/audioplayers-all-non-framework-target-headers.hmap -Xcc -ivfsoverlay -Xcc

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/all-product-headers.yaml -Xcc -iquote -Xcc

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/audioplayers-project-headers.hmap -Xcc
                   -I/Users/bryanhunsberger/AppBreweryProjectsAll/app_brewery_projects_all/build/ios/Debug-iphonesimulator/audioplayers/include -Xcc

-I/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiop
                   layers.build/DerivedSources-normal/x86_64 -Xcc

-I/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiop
                   layers.build/DerivedSources/x86_64 -Xcc

-I/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiop
                   layers.build/DerivedSources -Xcc -DPOD_CONFIGURATION_DEBUG\=1 -Xcc -DDEBUG\=1 -Xcc -DCOCOAPODS\=1 -Xcc -ivfsoverlay -Xcc

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/unextended-module-overlay.yaml -target-sdk-version 14.2 -module-name audioplayers -o

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/Objects-normal/x86_64/NotificationsHandler.o
               Command CompileSwift failed with a nonzero exit code
           
               CompileSwift normal x86_64
/Users/bryanhunsberger/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/audioplayers-0.17.3/ios/Classes/SwiftAudioplayersPlugin.swift (in
               target 'audioplayers' from project 'Pods')
                   cd /Users/bryanhunsberger/AppBreweryProjectsAll/app_brewery_projects_all/ios/Pods
                   /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swift -frontend -c
                   /Users/bryanhunsberger/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/audioplayers-0.17.3/ios/Classes/NotificationsHandler.swift -primary-file
                   /Users/bryanhunsberger/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/audioplayers-0.17.3/ios/Classes/SwiftAudioplayersPlugin.swift
                   /Users/bryanhunsberger/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/audioplayers-0.17.3/ios/Classes/Utils.swift
                   /Users/bryanhunsberger/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/audioplayers-0.17.3/ios/Classes/WrappedMediaPlayer.swift -emit-module-path

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/Objects-normal/x86_64/SwiftAudioplayersPlugin\~partial.swiftmodule -emit-module-doc-path

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/Objects-normal/x86_64/SwiftAudioplayersPlugin\~partial.swiftdoc -emit-module-source-info-path

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/Objects-normal/x86_64/SwiftAudioplayersPlugin\~partial.swiftsourceinfo -serialize-diagnostics-path

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/Objects-normal/x86_64/SwiftAudioplayersPlugin.dia -emit-dependencies-path

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/Objects-normal/x86_64/SwiftAudioplayersPlugin.d -emit-reference-dependencies-path

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/Objects-normal/x86_64/SwiftAudioplayersPlugin.swiftdeps -target x86_64-apple-ios8.0-simulator -enable-objc-interop -sdk
                   /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.2.sdk -I
                   /Users/bryanhunsberger/AppBreweryProjectsAll/app_brewery_projects_all/build/ios/Debug-iphonesimulator/audioplayers -F
                   /Users/bryanhunsberger/AppBreweryProjectsAll/app_brewery_projects_all/build/ios/Debug-iphonesimulator/audioplayers -enable-testing -g
-import-underlying-module
                   -module-cache-path /Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/ModuleCache.noindex -swift-version 5 -enforce-exclusivity\=checked -Onone
-D DEBUG -D
                   COCOAPODS -serialize-debugging-options -Xcc -working-directory -Xcc /Users/bryanhunsberger/AppBreweryProjectsAll/app_brewery_projects_all/ios/Pods
                   -enable-anonymous-context-mangled-names -Xcc

-I/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiop
                   layers.build/swift-overrides.hmap -Xcc -iquote -Xcc

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/audioplayers-generated-files.hmap -Xcc

-I/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiop
                   layers.build/audioplayers-own-target-headers.hmap -Xcc

-I/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiop
                   layers.build/audioplayers-all-non-framework-target-headers.hmap -Xcc -ivfsoverlay -Xcc

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/all-product-headers.yaml -Xcc -iquote -Xcc

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/audioplayers-project-headers.hmap -Xcc
                   -I/Users/bryanhunsberger/AppBreweryProjectsAll/app_brewery_projects_all/build/ios/Debug-iphonesimulator/audioplayers/include -Xcc

-I/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiop
                   layers.build/DerivedSources-normal/x86_64 -Xcc

-I/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiop
                   layers.build/DerivedSources/x86_64 -Xcc

-I/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiop
                   layers.build/DerivedSources -Xcc -DPOD_CONFIGURATION_DEBUG\=1 -Xcc -DDEBUG\=1 -Xcc -DCOCOAPODS\=1 -Xcc -ivfsoverlay -Xcc

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/unextended-module-overlay.yaml -target-sdk-version 14.2 -module-name audioplayers -o

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/Objects-normal/x86_64/SwiftAudioplayersPlugin.o
               Command CompileSwift failed with a nonzero exit code
           
               CompileSwift normal x86_64 /Users/bryanhunsberger/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/audioplayers-0.17.3/ios/Classes/Utils.swift (in target
               'audioplayers' from project 'Pods')
                   cd /Users/bryanhunsberger/AppBreweryProjectsAll/app_brewery_projects_all/ios/Pods
                   /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swift -frontend -c
                   /Users/bryanhunsberger/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/audioplayers-0.17.3/ios/Classes/NotificationsHandler.swift
                   /Users/bryanhunsberger/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/audioplayers-0.17.3/ios/Classes/SwiftAudioplayersPlugin.swift -primary-file
                   /Users/bryanhunsberger/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/audioplayers-0.17.3/ios/Classes/Utils.swift
                   /Users/bryanhunsberger/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/audioplayers-0.17.3/ios/Classes/WrappedMediaPlayer.swift -emit-module-path

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/Objects-normal/x86_64/Utils\~partial.swiftmodule -emit-module-doc-path

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/Objects-normal/x86_64/Utils\~partial.swiftdoc -emit-module-source-info-path

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/Objects-normal/x86_64/Utils\~partial.swiftsourceinfo -serialize-diagnostics-path

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/Objects-normal/x86_64/Utils.dia -emit-dependencies-path

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/Objects-normal/x86_64/Utils.d -emit-reference-dependencies-path

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/Objects-normal/x86_64/Utils.swiftdeps -target x86_64-apple-ios8.0-simulator -enable-objc-interop -sdk
                   /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.2.sdk -I
                   /Users/bryanhunsberger/AppBreweryProjectsAll/app_brewery_projects_all/build/ios/Debug-iphonesimulator/audioplayers -F
                   /Users/bryanhunsberger/AppBreweryProjectsAll/app_brewery_projects_all/build/ios/Debug-iphonesimulator/audioplayers -enable-testing -g
-import-underlying-module
                   -module-cache-path /Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/ModuleCache.noindex -swift-version 5 -enforce-exclusivity\=checked -Onone
-D DEBUG -D
                   COCOAPODS -serialize-debugging-options -Xcc -working-directory -Xcc /Users/bryanhunsberger/AppBreweryProjectsAll/app_brewery_projects_all/ios/Pods
                   -enable-anonymous-context-mangled-names -Xcc

-I/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiop
                   layers.build/swift-overrides.hmap -Xcc -iquote -Xcc

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/audioplayers-generated-files.hmap -Xcc

-I/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiop
                   layers.build/audioplayers-own-target-headers.hmap -Xcc

-I/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiop
                   layers.build/audioplayers-all-non-framework-target-headers.hmap -Xcc -ivfsoverlay -Xcc

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/all-product-headers.yaml -Xcc -iquote -Xcc

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/audioplayers-project-headers.hmap -Xcc
                   -I/Users/bryanhunsberger/AppBreweryProjectsAll/app_brewery_projects_all/build/ios/Debug-iphonesimulator/audioplayers/include -Xcc

-I/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiop
                   layers.build/DerivedSources-normal/x86_64 -Xcc

-I/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiop
                   layers.build/DerivedSources/x86_64 -Xcc

-I/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiop
                   layers.build/DerivedSources -Xcc -DPOD_CONFIGURATION_DEBUG\=1 -Xcc -DDEBUG\=1 -Xcc -DCOCOAPODS\=1 -Xcc -ivfsoverlay -Xcc

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/unextended-module-overlay.yaml -target-sdk-version 14.2 -module-name audioplayers -o

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/Objects-normal/x86_64/Utils.o
               Command CompileSwift failed with a nonzero exit code
           
               CompileSwift normal x86_64
/Users/bryanhunsberger/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/audioplayers-0.17.3/ios/Classes/WrappedMediaPlayer.swift (in
               target 'audioplayers' from project 'Pods')
                   cd /Users/bryanhunsberger/AppBreweryProjectsAll/app_brewery_projects_all/ios/Pods
                   /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swift -frontend -c
                   /Users/bryanhunsberger/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/audioplayers-0.17.3/ios/Classes/NotificationsHandler.swift
                   /Users/bryanhunsberger/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/audioplayers-0.17.3/ios/Classes/SwiftAudioplayersPlugin.swift
                   /Users/bryanhunsberger/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/audioplayers-0.17.3/ios/Classes/Utils.swift -primary-file
                   /Users/bryanhunsberger/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/audioplayers-0.17.3/ios/Classes/WrappedMediaPlayer.swift -emit-module-path

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/Objects-normal/x86_64/WrappedMediaPlayer\~partial.swiftmodule -emit-module-doc-path

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/Objects-normal/x86_64/WrappedMediaPlayer\~partial.swiftdoc -emit-module-source-info-path

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/Objects-normal/x86_64/WrappedMediaPlayer\~partial.swiftsourceinfo -serialize-diagnostics-path

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/Objects-normal/x86_64/WrappedMediaPlayer.dia -emit-dependencies-path

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/Objects-normal/x86_64/WrappedMediaPlayer.d -emit-reference-dependencies-path

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/Objects-normal/x86_64/WrappedMediaPlayer.swiftdeps -target x86_64-apple-ios8.0-simulator -enable-objc-interop -sdk
                   /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.2.sdk -I
                   /Users/bryanhunsberger/AppBreweryProjectsAll/app_brewery_projects_all/build/ios/Debug-iphonesimulator/audioplayers -F
                   /Users/bryanhunsberger/AppBreweryProjectsAll/app_brewery_projects_all/build/ios/Debug-iphonesimulator/audioplayers -enable-testing -g
-import-underlying-module
                   -module-cache-path /Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/ModuleCache.noindex -swift-version 5 -enforce-exclusivity\=checked -Onone
-D DEBUG -D
                   COCOAPODS -serialize-debugging-options -Xcc -working-directory -Xcc /Users/bryanhunsberger/AppBreweryProjectsAll/app_brewery_projects_all/ios/Pods
                   -enable-anonymous-context-mangled-names -Xcc

-I/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiop
                   layers.build/swift-overrides.hmap -Xcc -iquote -Xcc

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/audioplayers-generated-files.hmap -Xcc

-I/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiop
                   layers.build/audioplayers-own-target-headers.hmap -Xcc

-I/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiop
                   layers.build/audioplayers-all-non-framework-target-headers.hmap -Xcc -ivfsoverlay -Xcc

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/all-product-headers.yaml -Xcc -iquote -Xcc

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/audioplayers-project-headers.hmap -Xcc
                   -I/Users/bryanhunsberger/AppBreweryProjectsAll/app_brewery_projects_all/build/ios/Debug-iphonesimulator/audioplayers/include -Xcc

-I/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiop
                   layers.build/DerivedSources-normal/x86_64 -Xcc

-I/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiop
                   layers.build/DerivedSources/x86_64 -Xcc

-I/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiop
                   layers.build/DerivedSources -Xcc -DPOD_CONFIGURATION_DEBUG\=1 -Xcc -DDEBUG\=1 -Xcc -DCOCOAPODS\=1 -Xcc -ivfsoverlay -Xcc

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/unextended-module-overlay.yaml -target-sdk-version 14.2 -module-name audioplayers -o

/Users/bryanhunsberger/Library/Developer/Xcode/DerivedData/Runner-bziljagsqcdxznbwsduuhhjuhaaa/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/audiopla
                   yers.build/Objects-normal/x86_64/WrappedMediaPlayer.o
               Command CompileSwift failed with a nonzero exit code
           
               warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to
14.2.99. (in
               target 'Flutter' from project 'Pods')
               warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to
14.2.99. (in
               target 'audioplayers' from project 'Pods')
           
[  +54 ms] Could not build the application for the simulator.
[   +4 ms] Error launching application on iPhone 11.
[   +8 ms] "flutter run" took 34,130ms.
[  +10 ms] 
           #0      throwToolExit (package:flutter_tools/src/base/common.dart:14:3)
           #1      RunCommand.runCommand (package:flutter_tools/src/commands/run.dart:607:7)
           <asynchronous suspension>
           #2      FlutterCommand.verifyThenRunCommand (package:flutter_tools/src/runner/flutter_command.dart:977:18)
           #3      _rootRunUnary (dart:async/zone.dart:1198:47)
           #4      _CustomZone.runUnary (dart:async/zone.dart:1100:19)
           #5      _FutureListener.handleValue (dart:async/future_impl.dart:143:18)
           #6      Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:696:45)
           #7      Future._propagateToListeners (dart:async/future_impl.dart:725:32)
           #8      Future._completeWithValue (dart:async/future_impl.dart:529:5)
           #9      _AsyncAwaitCompleter.complete (dart:async-patch/async_patch.dart:40:15)
           #10     _completeOnAsyncReturn (dart:async-patch/async_patch.dart:311:13)
           #11     RunCommand.usageValues (package:flutter_tools/src/commands/run.dart)
           #12     _rootRunUnary (dart:async/zone.dart:1198:47)
           #13     _CustomZone.runUnary (dart:async/zone.dart:1100:19)
           #14     _FutureListener.handleValue (dart:async/future_impl.dart:143:18)
           #15     Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:696:45)
           #16     Future._propagateToListeners (dart:async/future_impl.dart:725:32)
           #17     Future._completeWithValue (dart:async/future_impl.dart:529:5)
           #18     Future._asyncCompleteWithValue.<anonymous closure> (dart:async/future_impl.dart:567:7)
           #19     _rootRun (dart:async/zone.dart:1190:13)
           #20     _CustomZone.run (dart:async/zone.dart:1093:19)
           #21     _CustomZone.runGuarded (dart:async/zone.dart:997:7)
           #22     _CustomZone.bindCallbackGuarded.<anonymous closure> (dart:async/zone.dart:1037:23)
           #23     _microtaskLoop (dart:async/schedule_microtask.dart:41:21)
           #24     _startMicrotaskLoop (dart:async/schedule_microtask.dart:50:5)
           #25     _runPendingImmediateCallback (dart:isolate-patch/isolate_patch.dart:118:13)
           #26     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:169:5)
           
           
[ +274 ms] ensureAnalyticsSent: 145ms
[   +2 ms] Running shutdown hooks
[        ] Shutdown hook priority 4
[   +6 ms] Shutdown hooks complete
[   +1 ms] exiting with code 1

```

```
Only unused imports...this is for a class
Analyzing app_brewery_projects_all...                                   

   info • The value of the field '_counter' isn't used • lib/i_am_rich/i_am_rich.dart:22:7 • unused_field
   info • The declaration '_incrementCounter' isn't referenced • lib/i_am_rich/i_am_rich.dart:24:8 • unused_element
   info • Unused import: 'package:app_brewery_projects/dicee/dicee.dart' • lib/main.dart:1:8 • unused_import
   info • Unused import: 'package:app_brewery_projects/xylophone/xylophone.dart' • lib/main.dart:3:8 • unused_import
   info • Unused import: 'i_am_rich/i_am_rich.dart' • lib/main.dart:6:8 • unused_import
   info • Unused import: 'magic_eight_ball/magic_eight_ball.dart' • lib/main.dart:7:8 • unused_import
   info • Unused import: 'package:flutter/cupertino.dart' • lib/quizzler/quiz_brain.dart:2:8 • unused_import
   info • Unused import: 'package:app_brewery_projects/quizzler/questions.dart' • lib/quizzler/quizzler.dart:1:8 • unused_import
   info • Duplicate import • lib/xylophone/xylophone.dart:3:8 • duplicate_import

9 issues found. (ran in 2.6s)
```

```
[✓] Flutter (Channel stable, 1.22.6, on macOS 11.2.1 20D74 darwin-x64, locale en-US)
    • Flutter version 1.22.6 at /Users/bryanhunsberger/Developer/flutter
    • Framework revision 9b2d32b605 (4 weeks ago), 2021-01-22 14:36:39 -0800
    • Engine revision 2f0af37152
    • Dart version 2.10.5

 
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.1)
    • Android SDK at /Users/bryanhunsberger/Library/Android/sdk
    • Platform android-30, build-tools 30.0.1
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 12.2)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.2, Build version 12B45b
    • CocoaPods version 1.10.1

[✓] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin installed
    • Dart plugin version 201.9317
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)

[✓] Connected device (1 available)
    • iPhone 11 (mobile) • 8EDC2968-4F29-4FF6-A81C-139C704307C3 • ios • com.apple.CoreSimulator.SimRuntime.iOS-14-2 (simulator)

• No issues found!
```

</details>

**Comment by jbryanh** [comment]
Could not submit entire verbose log for length restriction, so I just included from path_provider reference down.  If you want it, I can send.

In addition, following these steps resolves for me:
cd ios
1. pod deintegrate
2. rm Flutter/Flutter.podspec
3. rm podfile.lock
4. flutter clean
5. flutter run

This issue is repeating itself constantly. I'm not switching channels, however, it happens after I've switched projects in Android Studio.

**Comment by TahaTesser** [comment]
Hi @jbryanh 
Do you have consistent step reproducible steps? How to get this error when running the project?
I tried several times and unable to get this

<details>
<summary>flutter doctor -v</summary>

```bash
[✓] Flutter (Channel stable, 1.22.6, on macOS 11.2.1 20D74 darwin-x64, locale en-GB)
    • Flutter version 1.22.6 at /Users/tahatesser/Code/flutter_stable
    • Framework revision 9b2d32b605 (4 weeks ago), 2021-01-22 14:36:39 -0800
    • Engine revision 2f0af37152
    • Dart version 2.10.5
[✗] Android toolchain - develop for Android devices
    ✗ Unable to locate Android SDK.
      Install Android Studio from: https://developer.android.com/studio/index.html
      On first launch it will assist you in installing the Android SDK components.
      (or visit https://flutter.dev/docs/get-started/install/macos#android-setup for detailed instructions).
      If the Android SDK has been installed to a custom location, set ANDROID_SDK_ROOT to that location.
      You may also want to add it to your PATH environment variable.
[!] Xcode - develop for iOS and macOS (Xcode 12.4)
    • Xcode at /Volumes/Extreme/Xcode.app/Contents/Developer
    • Xcode 12.4, Build version 12D4e
    ✗ CocoaPods not installed.
        CocoaPods is used to retrieve the iOS and macOS platform side's plugin code that responds to your plugin usage on the Dart side.
        Without CocoaPods, plugins will not work on iOS or macOS.
        For more info, see https://flutter.dev/platform-plugins
      To install:
        sudo gem install cocoapods
[!] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
[✓] VS Code (version 1.53.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.19.0
[✓] Connected device (1 available)
    • Taha’s iPhone (mobile) • 00008020-001059882212002E • ios • iOS 14.4
! Doctor found issues in 3 categories.
```

</details>


Thank you

**Comment by jbryanh** [comment]
@TahaTesser, I'm sorry to say that I haven't noticed a clear trend that allows me to reproduce it consistently myself. At one point I thought it was related to changing channels, but lately it's been happening regardless of that.

And I'm under a time crunch, so I can't invest time to try to reproduce and provide firm steps. If enough users experience the issue, hopefully a trend develops.

I can say however that the following are pre-requisites or possible pre-requisites:

1. This only occurs after a project change for me.
2. I might be able to say that it occurs only after the previous project had completed an ios build.  (and app store upload)  And my manner of closing is to close xcode, then close android studio.
3.  It is not a pre-requisite to change channels.
4. I appears to occur when I go from a project with many plugins to a project with very few plugins.
5. I have experienced it the last few times after a change of day.  However, that is coincidental probably, and related to my habit of building archives at night on my production app, and starting my day on my class project lessons.

Sorry for the lack of facts.  I understand the facts and information you need to debug, they just come in conflict with real world time constraints.  So sorry.  There is a consistent fix for me, so the importance of this probably relates to the amount of interest that this continues to generate...however I expect it to be small, as the solution is quick and easy, just requires new users to investigate to solve.

**Comment by jmagman** [comment]
> Could not submit entire verbose log for length restriction, so I just included from path_provider reference down. If you want it, I can send.

You should be able to save to a file and attach the file to this issue.  I'd like to see the full Xcode build logs with the linker flags.

**Comment by jbryanh** [comment]
[Flutterhissue.pdf](https://github.com/flutter/flutter/files/6025140/Flutterhissue.pdf)

**Comment by jmagman** [comment]
Fortunately the actual linker issue will go away once https://github.com/flutter/flutter/pull/70224 reaches stable, and you upgrade.  You could try switching to `beta` and staying there for all your projects.

I don't understand how your projects are cross-contaminating though.  If project A has a plugin and project B doesn't, you shouldn't have any reference to the plugin in project B regardless of build order.

So based on your report `/Users/bryanhunsberger/AppBreweryProjectsAll/app_brewery_projects_all/` is the project that is failing, but has no reference to `path_provider`?  Can you attach your `pubspec.yaml` and `pubspec.lock` for that project?

**Comment by jbryanh** [comment]
pubspec.yaml
name: app_brewery_projects
description: Compilation of App Brewery Course Projects

publish_to: 'none' # Remove this line if you wish to publish to pub.dev

version: 1.0.0+1

environment:
  sdk: ">=2.7.0 <3.0.0"

dependencies:
  flutter:
    sdk: flutter

  cupertino_icons: ^1.0.0
  audioplayers: ^0.17.3
  rflutter_alert: ^1.1.0
  font_awesome_flutter: ^8.11.0

dev_dependencies:
  flutter_test:
    sdk: flutter

flutter:

  uses-material-design: true

  assets:
    - assets/images/
    - assets/sounds/
    - assets/

**Comment by jbryanh** [comment]
Pubspec.lock

# Generated by pub
# See https://dart.dev/tools/pub/glossary#lockfile
packages:
  async:
    dependency: transitive
    description:
      name: async
      url: "https://pub.dartlang.org"
    source: hosted
    version: "2.5.0-nullsafety.1"
  audioplayers:
    dependency: "direct main"
    description:
      name: audioplayers
      url: "https://pub.dartlang.org"
    source: hosted
    version: "0.17.3"
  boolean_selector:
    dependency: transitive
    description:
      name: boolean_selector
      url: "https://pub.dartlang.org"
    source: hosted
    version: "2.1.0-nullsafety.1"
  characters:
    dependency: transitive
    description:
      name: characters
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.1.0-nullsafety.3"
  charcode:
    dependency: transitive
    description:
      name: charcode
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.2.0-nullsafety.1"
  clock:
    dependency: transitive
    description:
      name: clock
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.1.0-nullsafety.1"
  collection:
    dependency: transitive
    description:
      name: collection
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.15.0-nullsafety.3"
  convert:
    dependency: transitive
    description:
      name: convert
      url: "https://pub.dartlang.org"
    source: hosted
    version: "2.1.1"
  crypto:
    dependency: transitive
    description:
      name: crypto
      url: "https://pub.dartlang.org"
    source: hosted
    version: "2.1.5"
  cupertino_icons:
    dependency: "direct main"
    description:
      name: cupertino_icons
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.0.0"
  fake_async:
    dependency: transitive
    description:
      name: fake_async
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.2.0-nullsafety.1"
  ffi:
    dependency: transitive
    description:
      name: ffi
      url: "https://pub.dartlang.org"
    source: hosted
    version: "0.1.3"
  file:
    dependency: transitive
    description:
      name: file
      url: "https://pub.dartlang.org"
    source: hosted
    version: "5.2.1"
  flutter:
    dependency: "direct main"
    description: flutter
    source: sdk
    version: "0.0.0"
  flutter_test:
    dependency: "direct dev"
    description: flutter
    source: sdk
    version: "0.0.0"
  flutter_web_plugins:
    dependency: transitive
    description: flutter
    source: sdk
    version: "0.0.0"
  font_awesome_flutter:
    dependency: "direct main"
    description:
      name: font_awesome_flutter
      url: "https://pub.dartlang.org"
    source: hosted
    version: "8.11.0"
  intl:
    dependency: transitive
    description:
      name: intl
      url: "https://pub.dartlang.org"
    source: hosted
    version: "0.16.1"
  matcher:
    dependency: transitive
    description:
      name: matcher
      url: "https://pub.dartlang.org"
    source: hosted
    version: "0.12.10-nullsafety.1"
  meta:
    dependency: transitive
    description:
      name: meta
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.3.0-nullsafety.3"
  path:
    dependency: transitive
    description:
      name: path
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.8.0-nullsafety.1"
  path_provider:
    dependency: transitive
    description:
      name: path_provider
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.6.27"
  path_provider_linux:
    dependency: transitive
    description:
      name: path_provider_linux
      url: "https://pub.dartlang.org"
    source: hosted
    version: "0.0.1+2"
  path_provider_macos:
    dependency: transitive
    description:
      name: path_provider_macos
      url: "https://pub.dartlang.org"
    source: hosted
    version: "0.0.4+8"
  path_provider_platform_interface:
    dependency: transitive
    description:
      name: path_provider_platform_interface
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.0.4"
  path_provider_windows:
    dependency: transitive
    description:
      name: path_provider_windows
      url: "https://pub.dartlang.org"
    source: hosted
    version: "0.0.4+3"
  platform:
    dependency: transitive
    description:
      name: platform
      url: "https://pub.dartlang.org"
    source: hosted
    version: "2.2.1"
  plugin_platform_interface:
    dependency: transitive
    description:
      name: plugin_platform_interface
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.0.3"
  process:
    dependency: transitive
    description:
      name: process
      url: "https://pub.dartlang.org"
    source: hosted
    version: "3.0.13"
  rflutter_alert:
    dependency: "direct main"
    description:
      name: rflutter_alert
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.1.0"
  sky_engine:
    dependency: transitive
    description: flutter
    source: sdk
    version: "0.0.99"
  source_span:
    dependency: transitive
    description:
      name: source_span
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.8.0-nullsafety.2"
  stack_trace:
    dependency: transitive
    description:
      name: stack_trace
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.10.0-nullsafety.1"
  stream_channel:
    dependency: transitive
    description:
      name: stream_channel
      url: "https://pub.dartlang.org"
    source: hosted
    version: "2.1.0-nullsafety.1"
  string_scanner:
    dependency: transitive
    description:
      name: string_scanner
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.1.0-nullsafety.1"
  term_glyph:
    dependency: transitive
    description:
      name: term_glyph
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.2.0-nullsafety.1"
  test_api:
    dependency: transitive
    description:
      name: test_api
      url: "https://pub.dartlang.org"
    source: hosted
    version: "0.2.19-nullsafety.2"
  typed_data:
    dependency: transitive
    description:
      name: typed_data
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.3.0-nullsafety.3"
  uuid:
    dependency: transitive
    description:
      name: uuid
      url: "https://pub.dartlang.org"
    source: hosted
    version: "2.2.2"
  vector_math:
    dependency: transitive
    description:
      name: vector_math
      url: "https://pub.dartlang.org"
    source: hosted
    version: "2.1.0-nullsafety.3"
  win32:
    dependency: transitive
    description:
      name: win32
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.7.4+1"
  xdg_directories:
    dependency: transitive
    description:
      name: xdg_directories
      url: "https://pub.dartlang.org"
    source: hosted
    version: "0.1.2"
sdks:
  dart: ">=2.10.0-110 <2.11.0"
  flutter: ">=1.12.13+hotfix.5 <2.0.0"

**Comment by TahaTesser** [comment]
Hi @jbryanh 
If the problem persists, please provide your `flutter doctor -v` with the  update
Thank you

**Comment by jmagman** [comment]
@jbryanh  If you're still hitting this with Flutter 2 (which contains the fix #70224) please let us know.  Attach your `ios/Podfile`.

**Comment by tuemaytinh1981** [comment]
path_provider-2.0.1 error when build with iOs, ---: url_launcher: ^6.0.3 ... error build with iOs newest.In file included from /Users/maitrongtue/.pub-cache/hosted/pub.dartlang.org/path_provider-2.0.1/ios/Classes/FLTPathProviderPlugin.m:5:
    /Users/maitrongtue/.pub-cache/hosted/pub.dartlang.org/path_provider-2.0.1/ios/Classes/FLTPathProviderPlugin.h:5:9: fatal error: 'Flutter/Flutter.h' file not found
    #import <Flutter/Flutter.h>
            ^~~~~~~~~~~~~~~~~~~
    1 error generated.
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description

**Comment by jmagman** [comment]
@tuemaytinh1981 Please file a new issue and completely fill out the template. Include your ios/Podfile.

**Comment by lbjlsy** [comment]
I have same problem

**Comment by lbjlsy** [comment]
this is my ios/Podfile
```
# Uncomment this line to define a global platform for your project
# platform :ios, '9.0'

# CocoaPods analytics sends network stats synchronously affecting flutter build latency.
ENV['COCOAPODS_DISABLE_STATS'] = 'true'

project 'Runner', {
  'Debug' => :debug,
  'Profile' => :release,
  'Release' => :release,
}

def flutter_root
  generated_xcode_build_settings_path = File.expand_path(File.join('..', 'Flutter', 'Generated.xcconfig'), __FILE__)
  unless File.exist?(generated_xcode_build_settings_path)
    raise "#{generated_xcode_build_settings_path} must exist. If you're running pod install manually, make sure flutter pub get is executed first"
  end

  File.foreach(generated_xcode_build_settings_path) do |line|
    matches = line.match(/FLUTTER_ROOT\=(.*)/)
    return matches[1].strip if matches
  end
  raise "FLUTTER_ROOT not found in #{generated_xcode_build_settings_path}. Try deleting Generated.xcconfig, then run flutter pub get"
end

require File.expand_path(File.join('packages', 'flutter_tools', 'bin', 'podhelper'), flutter_root)

flutter_ios_podfile_setup

target 'Runner' do
  use_frameworks!
  use_modular_headers!

  flutter_install_all_ios_pods File.dirname(File.realpath(__FILE__))
end

post_install do |pi|
  pi.pods_project.targets.each do |t|
    t.build_configurations.each do |config|
      config.build_settings['IPHONEOS_DEPLOYMENT_TARGET'] = '10.0'
    end
  end
end

```

**Comment by jmagman** [comment]
@lbjlsy Please file a new issue.

**Comment by exeptionerror** [comment]
[All Possible Solution added [Solved] error: ‘Flutter/Flutter.h’ file not found when flutter run on iOS](https://fluttercorner.com/error-flutter-flutter-h-file-not-found-when-flutter-run-on-ios/)

**Comment by jmagman** [comment]
> [All Possible Solution added [Solved] error: ‘Flutter/Flutter.h’ file not found when flutter run on iOS](https://fluttercorner.com/error-flutter-flutter-h-file-not-found-when-flutter-run-on-ios/)

That isn't even close to "all possible solutions" 😄   
Solution 1 and 2 aren't relevant for the last few stable versions of Flutter. 

`fatal error: 'Flutter\Flutter.h' file not found` is a linker error that means the Flutter framework isn't found in the search path for either a plugin or the app itself (depending on which target is complaining).  It's sometimes because people open their Xcode project and change settings that would make any framework not found, like overriding the search paths, or the linker flags, or the architecture settings.  Other times it's because they open their Podfile and remove the lines that Flutter was relying on to put the framework in the right place for plugins (solution 4 in that link).

If you are seeing this problem:
> Please file a new issue.
